taui/src/components/draw-neighborhood-bounds.js

```diff
@@ -29,17 +33,16 @@ export default class DrawNeighborhoodBounds extends React.PureComponent {
         data={p.neighborhoods}
         idField='id'
         tooltip={(feature) => getTooltip(feature)}
+        tooltipClassName='map__tooltip'
```

Comment:
What is `tooltipClassName`? It doesn't seem to be getting read anywhere. The [property name recognized by react-tooltip](https://github.com/wwayne/react-tooltip/blob/a2e8af550ae750aae6a68d2fb3195e5ebc258190/README.md#options) is `className`. The values assigned to `tooltipClass` above are hard-coded.

Reply:
It's an undocumented prop in `react-leaflet-vectorgrid` that allows us to style the tooltip. The tooltips appear to be Leaflet tooltips rather than react-tooltips.